config.yaml

```
@@ -44,3 +44,8 @@ options:
     default: ""
     description: |
       BMC password to be used by the redfish collector.
+  redfish-disable:
+    type: boolean
+    default: false
+    description: |
+      Boolean flag to disable the Redfish collector. Defaults to enabled (if redfish is available).
```

Thanks for flipping the enable/disable logic, but now this description needs to be updated :D
```suggestion
      Boolean flag to force disabling the Redfish collector. The collector will normally be run if Redfish support is detected on the unit.
```